zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+     */
+    public boolean isAdministrator(String email) throws JWTVerificationException {
+        boolean result = true;
+        if (StringUtils.isEmpty(email)) {
+            warn().user(email)
+                    .log("request for isAdministrator unsuccessful empty email");
+            result = false;
+        } else {
+            try {
+                Session s = getSessionfromEmail(email);
+                if (!hasPermission(s, ADMIN_PERMISSIONS)) {
+                    warn().user(email)
+                            .log("request for admin privilages check unsuccessful");
+                    result = false;
+                }
+
+            } catch (Exception exception) {
+                warn().user(email)
+                        .log("request for admin privilages exception caught");
+                result = false;
+            }
+        }
+
+        return result;
+    }
+
+
+    /**
+     * @param collection the collection to check users against.
+     * @return
+     * @throws JWTVerificationException
+     */
+
+    @Override
+    public List<User> getCollectionAccessMapping(Collection collection) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    /**
+     * @return
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public boolean hasAdministrator() throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    /**
+     * @param email   the email of the user to permit the permission to.
+     * @param session the {@link Session} of the user granting the permission.
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public void addAdministrator(String email, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    /**
+     * @param email   the email of the user to remove the permission from.
+     * @param session the {@link Session} of the user revoking the permission.
+     * @throws JWTVerificationException
+     * @deprecated with dp-identity-api
+     */
+    @Override
+    public void removeAdministrator(String email, Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canEdit(Session session) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canEdit(String email) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
+
+    @Override
+    public boolean canEdit(User user) throws JWTVerificationException {
+        throw new JWTVerificationException(JWTPERMISSIONSSERVICE_ERROR);
+    }
```

Comment:
I could be wrong but I think we will need to implement all of the `canEdit` & `canView` methods as I am pretty sure these are needed internally within Zebedee.